src/frame/mod.rs

```diff
@@ -105,6 +107,15 @@ pub enum Request {
     Disconnect,
 }
 
+/// A Modbus request with slave included
+#[derive(Debug, Clone, PartialEq, Eq)]
```

Comment:
The feature flag is still missing. Please also note that you need to reference `SlaveId` with the full path and remove the `use` statement to avoid warnings.

Reply:
Even if the module is private, the feature flag is for documentation and separation of concerns.

Reply:
Sorry for being slow, I'm not very used to actually making open source contributions :sweat:  (especially in rust)